api/controllers/WebActionsController.js

```diff
@@ -238,11 +238,14 @@ module.exports = {
         // If available, attach the referralCode to the redirect URL
         if (referralCode.length > 0) {
           redirectUrl += `&referralCode=${referralCode}`;
-
+          // Get partner/campaign name to identify where a user signed up
+          let partnerOrCampaign = req.body.partner
```

Comment:
Oh sorry, I _just_ read the "Questions / Considerations" section. Yea, I'd prefer just the above ☝️